.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow will build a package using Maven and then publish it to GitHub packages when a release is created
+# For more information see: https://github.com/actions/setup-java#apache-maven-with-a-settings-path
+
+name: Maven Package and Publish
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
```

Comment:
Why are we using an obsolete version of Java?

Reply:
If it was just for us, I would've used 11 but I worry about companies (especially big ones) who may use this and they might still be on JDK1.8.  I can change it easily though if you feel strongly about it.

Reply:
Is it possible to publish multiple versions?

Reply:
I know for example with Scala, you can publish one for say 2.11 and one for 2.12 of Scala.